view_controllers/OBASearchResultsMapViewController.m

```diff
@@ -966,21 +966,40 @@ - (void) showOutOfRangeAlert {
     }
 }
 
-- (void) showLocationServicesAlert {
+#pragma mark UIAlertViewDelegate
+
+- (void)showLocationServicesAlert {
     self.navigationItem.leftBarButtonItem.enabled = NO;
 
     if (![self.appDelegate.modelDao hideFutureLocationWarnings]) {
-        [self.appDelegate.modelDao setHideFutureLocationWarnings:YES];
+        [self.appDelegate.modelDao setHideFutureLocationWarnings:TRUE];
```

Comment:
Worth noting there is at least a few other places in the code with `TRUE` or `FALSE`.